simulator/index.md

```diff
@@ -5,13 +5,15 @@ title: Simulator
 
 # Simulator
 
-The simulator is used for the virtual competition but can also be a useful tool for becoming more familiar with our API or planning out game strategies.
+The simulator is a useful development tool, it allows you to become familiar with our API and to get started coding straight away even if you have yet to construct your robot.
+It could also be used to test out game strategies before applying them in real life.
 
 ## Programming
 
 Programming for the simulator is very similar to [programming for your physical robot]({{ site.baseurl }}/programming/).
 
-The simulator provides a subset of the `sr.robot3` [Robot API]({{ site.baseurl }}/programming/robot_api/) which is used on the physical robots. As a result code which is written for the simulator should largely work on your robot and vice-versa.
+The simulator provides a subset of the `sr.robot3` [Robot API]({{ site.baseurl }}/programming/robot_api/) which is used on the physical robots.
+As a result code which is written for the simulator should largely work on your robot and vice-versa.
```

Comment:
There doesn't seem to be an actual change here? Did we really need to update the whitespace? If you feel strongly that we should format the markdown a particular way that's fine, but _please_ can we not do this sort of thing as part of PRs which also change the content? Doing so makes it much more work for reviewers both now and in the future, to see what the changes are.